egs/babel_multilang/s5/local/nnet3/run_tdnn_multilingual.sh

```diff
@@ -204,9 +204,16 @@ EOF
     echo " output-layer name=output-${lang_index} dim=$num_targets max-change=1.5"
   done >> $dir/configs/network.xconfig
 
+  cat <<EOF > $dir/configs/edits.config
+  rename-node old-name=output-0 new-name=output
+EOF
   steps/nnet3/xconfig_to_configs.py --xconfig-file $dir/configs/network.xconfig \
     --config-dir $dir/configs/ \
-    --nnet-edits="rename-node old-name=output-0 new-name=output"
+    --edits-config=$dir/configs/edits.config
+
+  cat <<EOF >> $dir/configs/vars
```

Comment:
I updated xconfig_to_config to accept edits-config instead of nnet-edit, so this script should be updated.